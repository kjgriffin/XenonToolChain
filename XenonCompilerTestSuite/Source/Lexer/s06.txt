this is a test // of comments
that// are inline
/* multiline comment on a single line */
// /*multiline start
/* multiline comment
keeps going on and 
on and
*/ ends here
// and start at a line
and keep going