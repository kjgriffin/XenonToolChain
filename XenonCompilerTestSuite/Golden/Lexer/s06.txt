this
is
a
test
that
ends
here
and
keep
going